src/main/java/org/openrewrite/java/recipes/CorrectlySpacedDescriptions.java

```diff
@@ -0,0 +1,167 @@
+                        @Override
+                        public J.Binary visitBinary(J.Binary binary, ExecutionContext ctx) {
+                            J.Binary b = super.visitBinary(binary, ctx);
+                            // The left prefix is only modified when it's the first element of the J.Binary
+                            if (b.getLeft() instanceof J.Literal) {
+                                b = maybeFormatLeftPrefix(b);
+                            }
+                            b = maybeFormatLeftSuffix(b);
+                            b = maybeFormatRightPrefix(b);
+                            // The right suffix is only modified when it's tha last element of the J.Binary
+                            if (getCursor().getParentTreeCursor().getValue() instanceof J.Return) {
+                                b = maybeFormatRightSuffix(b);
+                            }
+                            return b;
+                        }
+                    }.visitNonNull(method, ctx, getCursor().getParentOrThrow());
+                }
+                return super.visitMethodDeclaration(method, ctx);
+            }
+
+            private J.Binary maybeFormatLeftPrefix(J.Binary b) {
+                return b.withLeft(maybeFormatLiteralPrefix((J.Literal) b.getLeft()));
+            }
+
+            private J.Binary maybeFormatLeftSuffix(J.Binary b) {
+                if (b.getLeft() instanceof J.Literal && b.getRight() instanceof J.Literal) {
+                    return b.withLeft(maybeFormatLiteralSuffix((J.Literal) b.getLeft()));
+                }
+                if (b.getLeft() instanceof J.Binary && b.getRight() instanceof J.Literal) {
+                    J.Binary left = (J.Binary) b.getLeft();
+                    Expression right = left.getRight();
+                    if (right instanceof J.Literal) {
+                        return b.withLeft(left.withRight(maybeFormatLiteralSuffix((J.Literal) right)));
+                    }
+                }
+                return b;
+            }
+
+            private J.Binary maybeFormatRightPrefix(J.Binary b) {
+                if (b.getLeft() instanceof J.Literal ||
+                      (b.getLeft() instanceof J.Binary &&
+                      ((J.Binary)b.getLeft()).getRight() instanceof J.Literal) &&
+                      b.getRight() instanceof J.Literal) {
+                    return b.withRight(maybeFormatLiteralPrefix((J.Literal) b.getRight()));
+                }
+                return b;
+            }
+
+            private J.Binary maybeFormatRightSuffix(J.Binary b) {
+                if (b.getRight() instanceof J.Literal) {
+                    return b.withRight(maybeFormatLiteralSuffix((J.Literal) b.getRight(), true));
+                }
+                return b;
+            }
+
+            private J.Literal maybeFormatLiteralPrefix(J.Literal l) {
+                if (l.getValue() instanceof String) {
```

Comment:
Though this achieves what you want it may be neater to check the literal type against `Primitive.String`